app/pb_kits/playbook/pb_typeahead/components/MultiValue.jsx

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+
+import React from 'react'
+import { components } from 'react-select'
+
+import { FormPill } from '../../'
+
+type Props = {
+  data: object,
+  multiValueTemplate: any,
+  removeProps: any,
+}
+
+const MultiValue = (props: Props) => {
+  const {
+    data,
+    removeProps,
+  } = props
+
+  const handleOnClick = removeProps.onClick
+
+  return (
+    <components.MultiValueContainer
+        className="text_input_multivalue_container"
+        {...props}
+    >
+      <If condition={data.avatarUrl}>
+        <FormPill
+            avatarUrl={data.avatarUrl}
+            marginRight="xs"
+            name={data.label}
+            onClick={handleOnClick}
+        />
+        <Else />
```

Comment:
The `Else` indentation is a little bit off.

Reply:
Yeh I formatted this for the syntax highlighter in my VSCode, but I don't like it either. It's just odd looking to me.